content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -76,31 +75,79 @@ To combine an event, click the three dots next to an event and select **+ Combin
+
+{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_refine_conversion.png" alt="Specify which facet you'd like to use to refine your conversion. The available options are Session, User, and Account." style="width:50%;" >}}
+ 
+
 
-You can measure conversion by the following attributes:
 
-- **Conversion count** - A count of users who went through the funnel you've defined.
-- **Conversion rate** - A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
+You can measure conversion by the following attributes:
 
-  **Note**: Only completed RUM sessions contribute to conversion rates.
+- **Conversion count**: A count of users who went through the funnel you've defined.
+- **Conversion rate**: A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
+- **Time to convert**: The time it took for the user to complete the step events. This is not available for the _Steps_ visualization, it changes to timeseries when selected.
 
-- **Time to convert** - The time it took for the user to complete the step events.
 
 You can measure these attributes **across all steps** or between **specific steps**.
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_conversion.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
 
-Use the **filter** selector to filter by various criteria that you define.
+Use the **filter** selector to scope to the various criteria you want to define. These filters are applied to all steps in your funnel.
 
 Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
 
+## Conversion computing metrics
+
+### How Datadog computes conversion metrics
+Consider a funnel with events `A → B → C` and event steps: 
+
+```
+A, A, A, B, C, C
+```
+
+In this case, Datadog counts one conversion. This is because the conversion calculations matches only the first occurence of event **A** and the first occurence of event **C** in the sequence. 
+
+<div class="alert alert-info">
+To further illustrate, if the user performs the event sequence <code>A, A, A, B, C, C, A, B, C...</code>, Datadog counts two conversions. The first conversion completes with the sequence <code>A, A, A, B, C, C</code>, and the second conversion begins with the next occurence of <code>A</code>.
+</div>
+
+The average time between steps is calculated based on the average duration between the first and last step of each conversion. This calculation takes all conversions into account regardless of whether you selected `unique` or `total` for the counts.
```

Comment:
```suggestion
Datadog calculates the average time between steps based on the average duration between the first and last step of each conversion.
```
I would move the thing about selecting Unique or Total for the counts to the next section. You haven't explained it yet, so your user doesn't know what you're talking about yet! And they could get confused because the past tense implied that they'd already done it.

(Also, to bring this back to what I was saying earlier about formatting - notice how in the next section it's formatted in bold, rather than monospace, which in my mind would be more appropriate for something a user has to type into a terminal, for example.)